src/axolotl/integrations/liger/__init__.py

```diff
@@ -24,6 +24,8 @@
 from liger_kernel.transformers.geglu import LigerGEGLUMLP
 from liger_kernel.transformers.model.llama import lce_forward
 from liger_kernel.transformers.model.qwen2 import lce_forward as qwen2_lce_forward
+from liger_kernel.transformers.model.mistral import lce_forward as mistral_lce_forward
+from liger_kernel.transformers.model.gemma import lce_forward as gemma_lce_forward
```

Comment:
please move these down into the various elif clauses below

otherwise the plugin will fail to load on older versions of the liger-kernel dependency, even if the user isn't necessarily dealing with mistral or gemma models themselves

cc #1876

Reply:
I went ahead and moved these.